apps/asap-server/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "compilerOptions": {
     "rootDir": ".",
     "resolveJsonModule": true,
-    "outDir": "build"
+    "outDir": "build",
+    "module": "commonjs",
```

Comment:
Does the bundler use actual TSC to output files? o.O

Reply:
Wouldn't it be better and faster to still bundle from `build`? We've already done the TS=>JS transpilation (and downleveling to current Node version instead of hardcoded es2019)?